boards/DCM/Src/App/states/App_FaultState.c

```diff
@@ -28,6 +29,14 @@ static void FaultStateRunOnEntry(struct StateMachine *const state_machine)
 
 static void FaultStateRunOnTick1Hz(struct StateMachine *const state_machine)
 {
+    struct DcmWorld *world = App_SharedStateMachine_GetWorld(state_machine);
+
+    // Clear inverter fault if requested by a PCAN node
+    App_SendNonPeriodicCanSignals_ClearInverterFaults(world);
+
+    // Open or close the inverter LV switches if requested by a PCAN node
+    App_SharedStates_ConfigInverterSwitches(world);
```

Comment:
nit; I understand that there is an if statement within the fxn here, but reading through this seems kinda weird bc we are in a 1Hz periodic task. Yet we are calling a fxn with `SendNonPeriodicCanSignals` in its name

Reply:
It seems as if we are periodically clearing the inverter faults. I think its nicer to read if we implement functions in `App_SendNonPeriodicCanSignals` directly within this state machine

Reply:
Same applies for config inverter switches

Reply:
> It seems as if we are periodically clearing the inverter faults. I think its a lot more clear if we implement functions in `App_SendNonPeriodicCanSignals` directly within this state machine (remove `App_SendNonPeriodicCanSignals` files), so we can see that these CAN msgs are only triggered via PCAN.
> 
> They are called once in this state machine anyways so no need for us to maintain separate `App_SendNonPeriodicCanSignal.c/h` files

Both `App_SendNonPeriodicCanSignals_ClearInverterFaults()` and `App_SharedStates_ConfigInverterSwitches()` are being called from within the fault and the init state. Hence, I decided to create functions for them.

Ideally, I'd have an event like an interrupt that would trigger when a `CLEAR_INV_FAULT_CMD` message is sent from PCAN. The handler would then simply send the param command message to one or both inverters. Since we have this periodic task structure set up nicely, I thought it'd be easier to simply check within a task if the `CLEAR_INV_FAULT_CMD` message has been sent. If so, have the DCM send the param command message and then clear the rx table to mark that event as handled.

I'll take out the `App_SendNonPeriodicCanSignals` files and will rename `App_SendNonPeriodicCanSignals_ClearInverterFaults` to `App_SharedStates_HandleClearInvFaultCmd` and place it in the appropriate files. I think this makes most sense.

Reply:
ah if we use it in two states then its cool

Reply:
I suggest implementing the if statement in the state machine instead of doing the check in `App_SharedStates_HandleClearInvFaultCmd`

```
if (App_CanRx_CLEAR_INV_FAULT_CMD_GetSignal_CLEAR_LEFT_FAULT(can_rx))
{
        App_SharedStates_HandleClearInvFaultCmd();
}
```

Reply:
^ actually ignore this